src/main/java/org/mule/tools/maven/repository/ClassLoaderModelAssembler.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.maven.repository;
+
+import static java.lang.String.format;
+import static org.mule.maven.client.internal.AetherMavenClient.MULE_PLUGIN_CLASSIFIER;
+import static org.mule.maven.client.internal.util.MavenUtils.getPomModelFromFile;
+import static org.mule.tools.maven.dependency.util.DependencyUtils.*;
+import static org.mule.tools.maven.dependency.util.DependencyUtils.toDependencies;
+import static org.mule.tools.maven.dependency.util.DependencyUtils.toDependency;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.artifact.Artifact;
+import org.apache.maven.artifact.DefaultArtifact;
+import org.apache.maven.artifact.handler.DefaultArtifactHandler;
+import org.apache.maven.artifact.repository.ArtifactRepository;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.eclipse.aether.graph.DependencyFilter;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.mule.maven.client.api.BundleDescriptorCreationException;
+import org.mule.maven.client.api.PomFileSupplierFactory;
+import org.mule.maven.client.api.model.BundleDependency;
+import org.mule.maven.client.api.model.BundleDescriptor;
+import org.mule.maven.client.internal.AetherMavenClient;
+import org.mule.tools.maven.dependency.model.ArtifactCoordinates;
+import org.mule.tools.maven.dependency.model.ClassLoaderModel;
+import org.mule.tools.maven.dependency.model.Dependency;
+import org.mule.tools.maven.dependency.util.DependencyUtils;
+
+
+public class ClassLoaderModelAssembler {
+
+  private static final String POM = "pom";
+  private static final String VERSION = "1.0.0";
```

Comment:
VERSION -> CLASS_LOADER_MODEL_VERSION
This way is a little bit more descriptive